src/ByteBuf.dfy

```diff
@@ -220,8 +220,8 @@ module ByteBuffer {
   method ByteCursorSplit(bc: ByteCursor, n: nat) returns (success: bool, bc0: ByteCursor, bc1: ByteCursor)  // aws_byte_cursor_advance
     requires GoodByteCursor(bc)
     ensures success ==>
-      GoodByteCursor(bc0) && bc0.a == bc.a && bc0.start == bc.start && bc0.len == n &&
-      ByteCursorAdvances(bc, bc1) && bc1.start == bc.start + n
+              GoodByteCursor(bc0) && bc0.a == bc.a && bc0.start == bc.start && bc0.len == n &&
+              ByteCursorAdvances(bc, bc1) && bc1.start == bc.start + n
```

Comment:
Why so much white space? I would have expected 2 spaces from the indentation of `ensures`?

Reply:
If you want 2 spaces from the indentation of `ensures`, `success` has to be on its own line.
In either way, the current rule for `==>` is that the consequence is indented by 2 spaces more than the hypothesis, to avoid visual ambiguity.
Moreover, the formatter can add these spaces for you so that you don't need to worry about writing them correctly on the first place.